sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/FunctionDefinition.java

```diff
@@ -19,32 +22,33 @@ public final class FunctionDefinition {
      */
     @Generated
     @JsonProperty(value = "name")
-    private String name;
```

Comment:
Since this is the generated code, manual changes at this parameter will be overwritten when we use TypeSpec to regenerate it next time.